README.md

```diff
@@ -84,7 +84,7 @@ If you are beginner and want to learn the basics of OCaml programming here is th
 - [ctypes](https://github.com/ocamllabs/ocaml-ctypes) – Library for binding to C libraries using pure OCaml.
 - [easy-format](https://github.com/mjambon/easy-format) – Pretty-printing library for OCaml.
 - [ocaml-rpc](https://github.com/mirage/ocaml-rpc) – Light library to deal with RPCs in OCaml.
-- [ocaml-containers](https://github.com/c-cube/ocaml-containers) – A small standard library extension, string library, and (in "misc") a bunch of random things of lower quality. BSD license.
+- [ocaml-containers](https://github.com/c-cube/ocaml-containers) – A lightweight,modular standard library extension, string library and interfaces to various libraries (bigarrays, unix, etc...). BSD license.
```

Comment:
No space after the comma! :scream:

Reply:
2d55731d6b2190bc9405559b8a8d7b55198ceb45 😛

Reply:
THANKS! :heart_eyes: